Misc/NEWS.d/next/Library/2019-07-09-19-38-26.bpo-3753.GX7s8S.rst

```diff
@@ -0,0 +1 @@
+Fix `regrtest` timeout for subprocesses.
```

Comment:
You should better explain the change. The change only impacts "python3 -m test -jN --timeout=TIMEOUT", it ensures that a worker process is stopped with a timeout if it runs longer than TIMEOUT seconds.